docs/user-guide/common-operations.md

```diff
@@ -17,4 +48,33 @@ The backend data can be wiped by using:
 kubehound backend wipe
 ```
 
-These commands will reboot backend services and wipe all data.
+!!! Warning
+    This command will **wipe ALL DATA and will not be recoverable**.
+
+## Dump
+
+### Create a dump localy with all needed k8s resources
+
+For instance, if you want to dump a configuration to analyse it later or just on another computer, KubeHound can create a self sufficient dump with the Kubernetes resources needed. By default it will create a `.tar.gz` file with all the dumper k8s resources needed.
+
+```bash
+kubehound dump local [directory to dump the data]
+```
+
+If for some reasons you need to have the raw data, you can add `--no-compress` flag to have a raw extract.
+
+!!! Note
```

Comment:
[See doc](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
```suggestion
> [!NOTE]
```

Reply:
nop, this is for the wiki, it is not the same. Run the `make local-wiki`